Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -908,29 +907,64 @@
 }
 
 #if PLATFORM(MAC)
+UnifiedPDFPlugin::ContextMenuItemTag UnifiedPDFPlugin::contextMenuItemTagFromDisplayMode(const PDFDocumentLayout::DisplayMode& displayMode) const
+{
+    switch (displayMode) {
+    case PDFDocumentLayout::DisplayMode::SinglePage: return ContextMenuItemTag::SinglePage;
+    case PDFDocumentLayout::DisplayMode::Continuous: return ContextMenuItemTag::SinglePageContinuous;
+    case PDFDocumentLayout::DisplayMode::TwoUp: return ContextMenuItemTag::TwoPages;
+    case PDFDocumentLayout::DisplayMode::TwoUpContinuous: return ContextMenuItemTag::TwoPagesContinuous;
+    }
+}
+
+PDFDocumentLayout::DisplayMode UnifiedPDFPlugin::displayModeFromContextMenuItemTag(const ContextMenuItemTag& tag) const
+{
+    ASSERT(tag == ContextMenuItemTag::SinglePage
+        || tag == ContextMenuItemTag::SinglePageContinuous
+        || tag == ContextMenuItemTag::TwoPages
+        || tag == ContextMenuItemTag::TwoPagesContinuous);
+
```

Comment:
Seems redundant with the `default` case

Reply:
Yeah I don't get it (I didn't add this, just moved and reformatted it). I'll get rid of it.